docs/data-access/data-download.md

```diff
@@ -90,10 +90,10 @@ There is no entry point or command defined for the image. The software is locate
 </TabItem>
 <TabItem value="Client-ENV">
 
-Download the **[latest version of the score-client](https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/score-client/[RELEASE]/score-client-[RELEASE]-dist.tar.gz)**. Once you have unzipped the tarball, change directories into the unzipped folder:
+Download the **[score-client](https://github.com/overture-stack/score/releases/download/5.11.0/score-client-5.11.0-dist.tar.gz)**. Once you have unzipped the tarball, change directories into the unzipped folder:
```

Comment:
use this instead? `https://github.com/overture-stack/score/releases/latest`

Reply:
quick research shows there's no easy way to do that... so let's move on for now :shipit:

Reply:
my concern is taht we're going to release a major score version update and our instructions wont match. Theres also a snapshot release there and we need to  review our automated release process to not promote in development versions... so for now I want to link to a specific version.